docs/developers/design_docs/cuda_backend.md

```diff
@@ -80,7 +80,6 @@ $ ../iree-build/iree/tools/iree-translate \
  -iree-input-type=mhlo \
  -iree-mlir-to-vm-bytecode-module \
  -iree-hal-target-backends=cuda \
- -iree-flow-dispatch-linalg-on-tensors \
```

Comment:
@ThomasRaoux looks like there may be other cleanup that could be done here after the linalg on tensors refactor

For example:
> Note that IREE had a legacy mode generating Linalg on Buffers. It is not supported by this path.

Reply:
Yes, good catch

Reply:
Actually, I'll create a PR to clean this up as several things are out of date.